examples/registration_form/lib/address/address_component.html

```diff
@@ -0,0 +1,42 @@
+<div ngControlGroup="address">
+    <material-input
+            type="text"
+            floatingLabel
+            label="Address 1"
+            ngControl="address1"
+            [(ngModel)]="address1"
```

Comment:
I don't think you actually need these ngModels... or at least they don't need to be two way you aren't doing anything with them in the component itself.

Reply:
I do eventually want to support a "default" value / editing existing data, so might add these back later. But agree that there's no value now.